src/manual.md

```diff
@@ -2,89 +2,91 @@
 
 [toc]
 
-## Disclaimer
+## Project Status / Disclaimer
 
-A word of caution: Deno is very much under development. We encourage brave early
-adopters, but expect bugs large and small. The API is subject to change without
-notice. [Bug reports](https://github.com/denoland/deno/issues) do help!
-
-## Introduction
+**A word of caution: Deno is very much under development.**
 
-A secure JavaScript/TypeScript runtime built with V8, Rust, and Tokio
-
-### Philosophy
-
-Deno aims to be a productive and secure scripting environment for the modern
-programmer.
-
-Deno will always be distributed as a single executable. Given a URL to a Deno
-program, it is runnable with nothing more than
-[the 10 megabyte zipped executable](https://github.com/denoland/deno/releases).
-Deno explicitly takes on the role of both runtime and package manager. It uses a
-standard browser-compatible protocol for loading modules: URLs.
-
-Deno provides security guarantees about how programs can access your system with
-the default being the most restrictive secure sandbox.
+We encourage brave early adopters, but expect bugs large and small. The API is subject to change without
+notice. [Bug reports](https://github.com/denoland/deno/issues) do help!
 
-Deno provides <a href="https://github.com/denoland/deno/tree/master/std">a set
-of reviewed (audited) standard modules</a> that are guaranteed to work with
-Deno.
+We are [actively working towards 1.0](https://github.com/denoland/deno/issues/2473), but there is no date guarantee.
 
-### Goals
+## Introduction
 
-- Support TypeScript out of the box.
+Deno is a JavaScript/TypeScript runtime with secure defaults and a great developer experience.
 
-- Uses "ES Modules" and does not support `require()`. Like the browser, allows
-  imports from URLs:
+It's built on V8, Rust, and Tokio.
```

Comment:
I think it's a little weird to call it a "secure runtime". "Secure defaults" feels more accurate and to the point. It leaves less room for argument.

I also added that it has a "great developer experience". I don't think this has historically been a first-class goal, but I think it's in alignment with the project and is part of where we provide value.